plaso/cli/helpers/artifact_definitions.py

```diff
@@ -127,6 +127,7 @@ def ParseOptions(cls, options, configuration_object):
             'Unable to read artifact definitions from: {0:s} with error: '
             '{1!s}').format(custom_artifacts_path, exception))
 
+    setattr(configuration_object, '_artifacts_registry', registry)
```

Comment:
This is not a configuration value I opt we solve this differently.

Reply:
Understood, just to be completely clear though, it was this way prior to my earlier artifact filters PR.  I removed it per your suggestion, but noticed today that it broke some pre-processing, I thought best to revert it back to original state, but can take a closer look now.  Specifically, 

      extraction_engine.PreprocessSources(
          self._artifacts_registry, self._source_path_specs,
          resolver_context=self._resolver_context)

in plaso/cli/extraction_tool.py

Reply:
Let's add a comment to change this then for now.

Reply:
Went ahead and moved this over to using the BuildArtifactsRegistry method, no longer passing registry in the configuration value.  Should be good now.